src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```diff
@@ -1316,4 +1325,4 @@ Outputs:
     Description: "The Kms Key Arn"
     Value: !GetAtt KMSKey.Arn
     Export:
-      Name: "KmsKeyArn"
+      Name: "KmsKeyArn"
```

Comment:
The file needs to have a newline character at the end of the file.
I think your editor might have removed this automatically.